Makefile

```diff
@@ -82,14 +82,20 @@ licenses: download ## Verifies dependency licenses
 verify: ## Verify code. Includes codegen, docgen, dependencies, linting, formatting, etc
 	go mod tidy
 	go generate ./...
-	hack/validation/kubelet.sh
-	hack/validation/taint.sh
-	hack/validation/requirements.sh
-	hack/validation/labels.sh
-	hack/validation/resources.sh
-	hack/validation/status.sh
-	cp -r pkg/apis/crds kwok/charts
-	hack/validation/kwok-requirements.sh
+	hack/validation/v1/kubelet.sh
```

Comment:
How many of these validations do we still need? What version did we need to get onto where we could build CEL expressions for these things rather than having to rely on this injection after the fact?

Reply:
> What version did we need to get onto

Is this indicating an offline conversation or a genuine question, because I have the same question :D